plugins/objectstore/networkservice_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectstore
+
+import (
+	"testing"
+
+	"github.com/ligato/networkservicemesh/netmesh/model/netmesh"
+)
+
+type networkserviceStore struct {
+	*networkServicesStore
+}
+
+const (
+	nsTestName      = "EndpointTest"
+	nsTestNamespace = "default"
+)
+
+var networkservices *networkserviceStore
+
+func TestNetworkServiceStore(t *testing.T) {
+	networkservices := &networkserviceStore{}
+	networkservices.networkServicesStore = newNetworkServicesStore()
+
+	ns := netmesh.NetworkService{
```

Comment:
I think line 38 and 42 should be combined.

Reply:
Done.